src/main/java/com/onionshop/entities/Circle.java

```diff
@@ -0,0 +1,17 @@
+package com.onionshop.entities;
+
+public class Circle extends Shape implements Tool {
```

Comment:
If shape implements tool, then Circle implements tool by default, you don't need to add this

Reply:
whoops forgot I think I added implements tool for circle & line & rectangle sorry bout that